resources/js/views/Dashboard.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="dashboard container-fluid">
+    <div v-if="this.user.group === 'control'">
```

Comment:
change `` this.user.group `` to `` this.user.user_group `` in this line as well as in any other instances where group is checked. This is because the backend is passing the group as `` user_group ``